content/en/docs/setup/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Setup
+description: Learn how to install and configure Falco
+aliases:
+- ../install-operate
```

Comment:
Ah yes, it's the same issue as gRPC and redirects to top level instead of `docs/install-operate/`